CMakeLists.txt

```diff
@@ -404,9 +404,19 @@ endif()
 
 file(MAKE_DIRECTORY ${ZMQ_BACKEND_DEST})
 
+if(NOT "${SKBUILD_SABI_COMPONENT}" STREQUAL "")
+  # set stable API
+  # assume we are targeting >= current Python version
+  # this isn't required, but we can't seem to get `wheel.py-api
+  message("SKBUILD_LIMITED_API=${SKBUILD_LIMITED_API}")
```

Comment:
It turns out that `SKBUILD_LIMITED_API` should have originally been documented as `SKBUILD_LIMITED_ABI`, then it was later removed in favor of checking if `SKBUILD_SABI_COMPONENT` is not empty. See https://github.com/scikit-build/scikit-build-core/pull/957/files#r1873869308